pkg/ometrics/go.mod

```
@@ -1,7 +1,7 @@
 module github.com/getoutreach/gobox/pkg/ometrics
 
 go 1.22
-toolchain go1.23.4
+toolchain go1.24.1
```

Revert toolchain change, this is a bug in Dependabot
```suggestion
toolchain go1.23.4
```